src/Stratis.Bitcoin/Controllers/NodeController.cs

```diff
@@ -119,7 +119,7 @@ public IActionResult Status()
                 Agent = this.connectionManager.ConnectionSettings.Agent,
                 ProcessId = Process.GetCurrentProcess().Id,
                 Network = this.fullNode.Network.Name,
-                ConsensusHeight = this.chainState.ConsensusTip.Height,
+                ConsensusHeight = this.chainState.ConsensusTip?.Height,
```

Comment:
Are you sure `chainState` can not also be null?

Reply:
`chainState` gets injected by DI into the constructor 👍